package.json

```diff
@@ -51,6 +51,7 @@
     "prettier-plugin-solidity": "^1.0.0-beta.1",
     "pretty-quick": "^2.0.1",
     "secp256k1": "^3.7.1",
+    "solc-0.8": "npm:solc@^0.8.4",
     "solidity-docgen": "^0.5.3",
```

Comment:
do we not also have to bump solidity-docgen?

Reply:
as discussed offline we will strip this from this PR with the goal of bringing back the dock gen in a later PR.